packages/frontend/components/Image.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+interface Image {
```

Comment:
You've to make sure the inteface name and component function name is different, e.g use this instead `ImageInterface` or anything else